rules/html.lua

```diff
@@ -21,7 +21,7 @@ local rspamd_regexp = require "rspamd_regexp"
 reconf['MIME_HTML_ONLY'] = {
   re = 'has_only_html_part()',
   score = 0.2,
-  description = 'Messages that have only HTML part',
+  description = 'Messages has only an HTML part',
```

Comment:
Is `message` supposed to be singular?

Reply:
```suggestion
  description = 'The message has only an HTML part',
```

Reply:
Missed that, thank you. Pushed a fix to this branch.